staging/src/k8s.io/cri-client/pkg/utils_test.go

```diff
@@ -102,7 +103,7 @@ func TestVerifySandboxStatus(t *testing.T) {
 		if actual != nil {
 			assert.EqualError(t, actual, status.expected.Error())
 		} else {
-			assert.Nil(t, status.expected)
+			require.NoError(t, status.expected)
```

Comment:
testifylint suggests to use `assert.NoError` here:
```
$ testifylint ./staging/src/k8s.io/cri-client/pkg/...
/home/ed/go/src/k8s.io/kubernetes/staging/src/k8s.io/cri-client/pkg/utils_test.go:105:4: error-nil: use assert.NoError
...
```
Why do you prefer to use `require.Noerror`

Reply:
> if you run it again, it will suggest to replace assert.NoError with require.NoError

It doesn't:
```
$ git diff
diff --git a/staging/src/k8s.io/cri-client/pkg/utils_test.go b/staging/src/k8s.io/cri-client/pkg/utils_test.go
index e007442d8fa..a5281ab94ba 100644
--- a/staging/src/k8s.io/cri-client/pkg/utils_test.go
+++ b/staging/src/k8s.io/cri-client/pkg/utils_test.go
@@ -102,7 +102,7 @@ func TestVerifySandboxStatus(t *testing.T) {
                if actual != nil {
                        assert.EqualError(t, actual, status.expected.Error())
                } else {
-                       assert.Nil(t, status.expected)
+                       assert.NoError(t, status.expected)
                }
        }
 }
@@ -175,7 +175,7 @@ func TestVerifyContainerStatus(t *testing.T) {
                if actual != nil {
                        assert.EqualError(t, actual, status.expected.Error())
                } else {
-                       assert.Nil(t, status.expected)
+                       assert.NoError(t, status.expected)
                }
        }
 }
```
$ testifylint ./staging/src/k8s.io/cri-client/pkg/ 
kubernetes/staging/src/k8s.io/cri-client/pkg/remote_image_test.go:70:2: require-error: for error assertions use require
kubernetes/staging/src/k8s.io/cri-client/pkg/remote_image_test.go:96:2: require-error: for error assertions use require
```

```